examples/gallery/demos/bokeh/mandelbrot_section.ipynb

```diff
@@ -59,18 +59,20 @@
    "source": [
     "dots = np.linspace(-0.45, 0.45, 19)\n",
     "fractal = hv.Image(array)\n",
+    "\n",
     "# First example on the old holoviews.org homepage was:\n",
     "# ((fractal * hv.HLine(y=0)).hist() + fractal.sample(y=0))\n",
     "layouts = {y: (fractal * hv.Points(fractal.sample([(i,y) for i in dots])) +\n",
     "               fractal.sample(y=y) +\n",
     "               hv.operation.threshold(fractal, level=np.percentile(fractal.sample(y=y)['z'], 90)) +\n",
     "               hv.operation.contours(fractal, levels=[np.percentile(fractal.sample(y=y)['z'], 60)]))\n",
-    "            for y in np.linspace(-0.3, 0.3, 11)} # Half the frames of the bokeh version\n",
+    "            for y in np.linspace(-0.3, 0.3, 21)}\n",
+    "\n",
+    "layout = hv.HoloMap(layouts, kdims='Y').collate()\n",
     "\n",
-    "composition = hv.HoloMap(layouts, kdims='Y').collate().cols(2)\n",
-    "composition.opts(\n",
-    "    opts.Contours(show_legend=False), \n",
-    "    opts.Points(scaling_factor=50))"
+    "layout.opts(\n",
+    "    opts.Contours(color='w', show_legend=False),\n",
+    "    opts.Points(size=dim('z')*10)).cols(2)"
```

Comment:
Not sure about putting `.cols` at the end. I'm not necessarily against it, but I've been avoiding doing this sort of thing to keep the styling rules simple and consistent.

Reply:
Okay, I don't object to moving it above.